docs/04-Captures/managed-storage-account.md

```diff
@@ -17,8 +17,15 @@ To simplify customers' work to decide where to store the packets and then downlo
 To enable the managed storage account, you need to specify the following configuration in the helm command,
 
 ```bash
---set operator.enabled=true \
---set operator.capture.enableManagedStorageAccount=true
+helm upgrade --install retina ./deploy/legacy/manifests/controller/helm/retina/ \
+  --set operator.enabled=true \
+  --set operator.capture.enableManagedStorageAccount=true \
+  --set capture.enableManagedStorageAccount=true \
```

Comment:
are there two values for managed SA? `operator.capture.enableManagedStorageAccount` and `capture.enableManagedStorageAccount`. What is the difference, and do we really need both?

Reply:
Thanks for catching this. It should be only capture.enableManagedStorageAccount=true because I have removed capture  section from operator to not make operator configuration lengthy considering we configure capture only on operator.
Removed the operator.capture.enableManagedStorageAccount